### ISSUE TITLE
PPO doesn't seem to learn

### ISSUE THREAD

**Issue by kapsl** [SurrealAI/surreal#17]
Hi,
I tried some robotic fitting task and it works pretty well with DDPG, but with PPO, it does not seem to learn anything. Is there some known issues, or some known hyperparameters that need to be adjusted?

Thanks!

**Comment by anchit** [comment]
Hi,
I assume you are training on low level states (not pixels).
Try changing these hyper-parameters in ppo_configs.py.

`action_repeat: 1`
`use_z_filter: True`
`limit_episode_length: 200`

Using these should essentially give you the same parameters we used in our paper.

**Comment by kapsl** [comment]
No luck so far. We have an own environment. With ddpg it is fine. Ppo shows no progress at all. When I have time I will play a little bit around with some hyper parameters. It's strange because the task should be really easy with only 2 action dimensions.

**Comment by LinxiFan** [comment]
We will address this issue after the holidays. Thanks for your patience!

**Comment by kapsl** [comment]
Ok I found that it just takes a very long time and more agents - compared to ddpg. Actually it takes about 45min with 7 agents till it is not getting worse but starting learning... I will try increasing the learning rate, maybe this helps?!

**Comment by LinxiFan** [comment]
We are uploading a set of hyperparameters tuned for local desktops soon. Thanks for your patience!
